spec/helpers/analytics_helpers_spec.rb

```diff
@@ -0,0 +1,179 @@
+require "spec_helper"
+require_relative "../../helpers/analytics_helpers"
+
+RSpec.describe AnalyticsHelpers do
+  let(:helper) { Class.new { extend AnalyticsHelpers } }
+
+  describe "#urlize" do
+    it "returns a url safe version of the given string" do
+      expect(helper.urlize("An event-name")).to eq("an_event_name")
+    end
+  end
+
+  describe "#build_event" do
+    it "returns an ordered hash given an array of un-ordered hashes" do
+      actual = [
```

Comment:
I would call this `input` instead of `actual`. I always think "actual" means "the actual result", which you'd get if you did

```
actual = helper.build_event([...
```

and then `expect(actual).to eq(expected)`